### DISCUSSION TITLE
Cannot connect to host localhost:9257 ssl:<ssl.SSLContext object at 0x7fbf40c63540> [Connect call failed ('127.0.0.1', 9257)]

### DISCUSSION THREAD

**Discussion by finndayton** [xch-gallery/nft-companion#9] [Q&A]
Hi Andreas!

I cloned you repo and set up the venv. I am getting the following error when running
`python3 nft.py create --name "Curly Nonchalant Marmot" --uri "https://example.com/curly-nonchalant-marmot.png"`

I am running a Chia full node on my laptop. Any advice where to start looking? Thanks!

Error: 
<img width="1080" alt="Screen Shot 2022-03-08 at 10 21 18 PM" src="https://user-images.githubusercontent.com/59621384/157384099-e6125911-7f1f-4700-98df-fc8cea969fc9.png">

**Comment by greimela** [comment]
The NFT companion is not interacting with the full node, but with the wallet.
So you need to have a wallet running on your machine.

With Chia 1.3 or the light wallet beta, the full node is not required at all.

**Reply by finndayton** [reply]
Where do I point xch-gallery code to my wallet, then?

**Reply by greimela** [reply]
Per default, it tries to find it on the same machine it is running on (`localhost` on port `9257`).
Right now, there is no easy way to interact with a wallet on a remote machine.

If you are interested, we will soon have a way more integrated NFT creation experience.
The current nft-companion is rather developer focused.

**Reply by aaronzr** [reply]
Hi Andreas, I'm on Finn's team and am also having trouble.  I have the Chia client running on my machine and it is using port 9257, but the code still will not connect.  It throws an SSL certificate error for me.
<img width="1046" alt="Screen Shot 2022-03-10 at 1 22 31 PM" src="https://user-images.githubusercontent.com/30187816/157739849-e5b69891-d051-4960-ba30-4f02f943059f.png">
<img width="1440" alt="Screen Shot 2022-03-10 at 1 28 13 PM" src="https://user-images.githubusercontent.com/30187816/157739862-3cc804c3-6985-42b1-811c-af85da3feda5.png">

<img width="1440" alt="Screen Shot 2022-03-10 at 1 28 10 PM" src="https://user-images.githubusercontent.com/30187816/157739858-28e2a1fa-6b1a-4144-a61c-9715a6e219c3.png">


Any ideas?
